build.sbt

```diff
@@ -494,6 +502,25 @@ lazy val bench = project.dependsOn(macrosJVM, coreJVM, freeJVM, lawsJVM)
   ))
   .enablePlugins(JmhPlugin)
 
+
+lazy val binCompactTest = project
```

Comment:
Should this be `binCompatTest` (and also in the source folder)?

Reply:
oops